### ISSUE TITLE
Multiple domains

### ISSUE THREAD

**Issue by raphaeltm** [DefangLabs/defang#247]
It would be useful to be able to specify multiple domains for a service. There are cases where it might be useful to set some context for an application based on the domain used to access it. Like, I can imagine a dashboard that uses the same app, but modifies some language or visibility of menu items based on the domain:

```
teacher.app.com
```

```
student.app.com
```

Similar but different to #246

**Comment by raphaeltm** [comment]
Related to #174

**Comment by lionello** [comment]
This also blocks Nounly.

**Comment by lionello** [comment]
Blocked by #523

**Comment by Prakash-Sundaresan** [comment]
Duplicate of https://github.com/orgs/DefangLabs/projects/4/views/20?pane=issue&itemId=67388462 ?

**Comment by Prakash-Sundaresan** [comment]
Duplicate of https://github.com/orgs/DefangLabs/projects/4/views/20?pane=issue&itemId=67388462 ?

**Comment by raphaeltm** [comment]
> Duplicate of https://github.com/orgs/DefangLabs/projects/4/views/20?pane=issue&itemId=67388462 ?

My understanding of the spec is that that issue would address internal hostname aliases, not public facing? But @lionello can correct me if I'm wrong.

**Comment by lionello** [comment]
Duplicate of https://github.com/DefangLabs/defang-mvp/issues/1108
